frontend/nuxt.config.js

```diff
@@ -60,4 +64,9 @@ export default {
   // Build Configuration: https://go.nuxtjs.dev/config-build
   build: {
   },
+
+  publicRuntimeConfig: {
+    umapaUrl: 'https://www.umapa.eu/embed-pro',
```

Comment:
tady potřebujeme url vč. filtrů , tj. https://www.umapa.eu/embed-pro?fcat=25972&attr54492=iynt,p0pw,1yll&_from=mistoveskole (trochu problém může být, že výsledek hledání je třeba vložit doprostřed místo embed-pro - skládání nechám na vás)

Reply:
Done